account_invoice_tax_required/README.rst

```diff
@@ -22,17 +35,19 @@ Contributors
 ------------
 
 * Vincent Renaville <vincent.renaville@camptocamp.com>
+* Angel Moya <odoo@tecnativa.com>
```

Comment:
I usually put unaccent, on my DNI (Spanish Identity Card) is writen unaccent, and also in all oficial documents. I know that it is wrong write on spanish, but unaccent is more international ;)